ocs_ci/deployment/deployment.py

```diff
@@ -643,3 +648,117 @@ def patch_default_sc_to_non_default(self):
                 f"-p {patch} "
                 f"--request-timeout=120s"
             )
+
+    def uninstall_osc(self):
```

Comment:
please add a description and also include the documentation link describing the procedure